lib/elixir_auth_google/mock.ex

```diff
@@ -1,19 +1,19 @@
 defmodule App.ElixirAuthGoogle.Mock do
   def get_token(_code, _conn) do
-    {:ok, %{"access_token" => "token"}}
+    {:ok, %{:access_token => "token"}}
   end
 
   def get_user_profile(_token) do
     {:ok,
      %{
-      "email" => "nelson@gmail.com",
-      "email_verified" => true,
-      "family_name" => "Correia",
-      "given_name" => "Nelson",
-      "locale" => "en",
-      "name" => "Nelson Correia",
-      "picture" => "https://lh3.googleusercontent.com/a-/AAuE7mApnYb260YC1JY7a",
-      "sub" => "940732358705212133793"
+      :email => "nelson@gmail.com",
```

Comment:
the Map+atom syntax is preferable:
```elixir
%{
email: "nelson@gmail.com",
email_verified: true,
family_name: "Correia",
given_name: "Nelson",
locale: "en",
name: "Nelson Correia",
picture: "https://lh3.googleusercontent.com/a-/AAuE7mApnYb260YC1JY7a",
sub: "940732358705212133793"
}
```

One of the biggest reasons _I_ want to use atoms as the keys in maps (_wherever possible_)
is that it makes Elixir Maps _much_ easier to read. 
Sadly they aren't _quite_ valid JSON but it makes them easier to validate.